### ISSUE TITLE
Cannot use single_category_selection

### ISSUE THREAD

**Issue by amb-jarek** [sulu/sulu#5295]
| Q | A
| --- | ---
| Sulu Version | 2.1.0-RC1
| Browser Version | Irrelevant

# Actual Behavior

As soon as I add `single_category_selection` property to my page template definition I'm getting following error:
```
Content type with alias "single_category_selection" has not been registered. 
Known content types are: "number", "text_line", "text_area", ...
```

This content type has been added in this PR: https://github.com/sulu/sulu/pull/5106
According to Sulu dev asked on Slack some kind of service is missing.

# Expected Behavior

I should be able to use `single_category_selection` content type

# Steps to Reproduce
Add following property to your page template definition: 
```xml
<property name="category" type="single_category_selection">
    <meta>
        <title lang="en">Single Category Selection</title>
    </meta>

    <params>
        <param name="request_parameters" type="collection">
            <param name="rootKey" value="my_category_key"/>
        </param>
    </params>
</property>
```

# Possible Solutions

Register service for `SingleCategorySelection` in `src/Sulu/Bundle/CategoryBundle/Resources/config/services.xml`

**Comment by drakulis** [comment]
It start working when I added following to my `services.yml`

```
    Sulu\Bundle\CategoryBundle\Content\Types:
        class: Sulu\Bundle\CategoryBundle\Content\Types\SingleCategorySelection
        tags: [{ name: 'sulu.content.type', alias: 'single_category_selection' }]
```

Thank you @luca-rath !

**Comment by amb-jarek** [comment]
It start working when I added following to my services.yml

    Sulu\Bundle\CategoryBundle\Content\Types:
        class: Sulu\Bundle\CategoryBundle\Content\Types\SingleCategorySelection
        tags: [{ name: 'sulu.content.type', alias: 'single_category_selection' }]

Thank you @luca-rath !

**Comment by danrot** [comment]
Fixed in #5297. Thanks for reporting!
